docs/UserGuide.md

```diff
@@ -17,15 +17,18 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `17` or above installed in your Computer.
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-F14a-1/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy the file to the folder you want to use as the _home folder_ for your address book.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+1. Open a command terminal, `cd` into the folder that you put the jar file in
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+1. Use the `java -jar ugteach.jar` command to run the application.<br><br>
+   A GUI similar to the image shown below should appear in a few seconds. Note how the app contains some sample data.
+   <br>
+   ![Ui](images/UGTeach.png)
+
+1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br><br>
```

Comment:
Might consider fixing the indexing

Reply:
I'm not very sure, but on markdown the indexing looks okay

Reply:
Yup but abit funky so might be better to change.

Reply:
You have a good point. I will change it this instant!